src/main/java/ch/njol/skript/classes/data/BukkitClasses.java

```diff
@@ -1836,5 +1837,31 @@ public String toVariableNameString(Attribute a) {
 					}
 				})
 				.serializer(new EnumSerializer<>(Attribute.class)));
+
+		EnumUtils<Environment> environments = new EnumUtils<>(Environment.class, "environments");
+		Classes.registerClass(new ClassInfo<>(Environment.class, "environment")
+				.user("(world ?)?environments?")
+				.name("Environment Type")
+				.description("Represents the type of a world")
```

Comment:
Since other types do this as well
```suggestion
				.description("Represents the type of a world.")
```